server/types.ts

```diff
@@ -80,8 +80,8 @@ export const createOAuthProvider = z.object({
   app: z.nativeEnum(Apps),
 })
 
-export const connectorIdSchema = z.object({
-  connectorId: z.number(),
+export const connectorExternalIdSchema = z.object({
+  connectorExternalId: z.string(),
```

Comment:
let's assume that what goes to frontend will be externalId by default so we can call it connectorId too